Listener/InstallerLocaleListener.php

```diff
@@ -21,7 +21,7 @@ class InstallerLocaleListener
 
     public function __construct(LanguageProvider $languageProvider)
     {
-        $this->installerLanguages = array_values($languageProvider->getSupportedLanguages());
+        $this->installerLanguages = \array_keys($languageProvider->getSupportedLanguages());
```

Comment:
`getSupportedLanguages()` already returns the array keys of `$supportedLanguages`.